subsys/shell/shell_vt100.h

```diff
@@ -18,6 +18,7 @@
 #define SHELL_VT100_ASCII_CTRL_F	(0x06)
 #define SHELL_VT100_ASCII_CTRL_K	(0x0B)
 #define SHELL_VT100_ASCII_CTRL_L	(0x0C)
+#define SHELL_VT100_ASCII_CTRL_S	(0x13)
```

Comment:
This is actually XOFF character, sent when you press Ctrl+S. You can send also XON when you press Ctrl-Q.
More info: https://en.wikipedia.org/wiki/Software_flow_control

Reply:
fixed